lib/screencap/raster.js

```diff
@@ -58,6 +61,11 @@ function pickupNamedArguments() {
     if(args.resourceWait)  { resourceWait = args.resourceWait; }
     if(args.maxRenderWait) { maxRenderWait = args.maxRenderWait; }
     if(args.cutoffWait)    { cutoffWait = args.cutoffWait; }
+    if(args.auth)          {
+      pair = args.auth.split(/:/);
```

Comment:
Might be nicer to just have separate username and password fields as this will have issues if either contains a ":"